draft-private-access-tokens.md

```diff
@@ -692,20 +660,30 @@ the Client was able to request a token for the given (CLIENT_ID, ORIGIN_NAME) tu
 its budget. Failure to present a token can be interpreted as a signal that the client's token
 budget was exceeded.
 
-## Client Linking
+## Client Geo-Location {#implement-geolocation}
 
-In this example, a site wishes to determine if multiple connections from a user are from
-one user, or multiple. It is not important to map a connection to a non-anonymous user identifier.
+To instantiate this use case, the Issuer has an issuing key pair per geographic region, i.e.,
+each region has a unique policy key. The Mediator forwards information about the Client's
```

Comment:
I don't know if the mediator would be sending the geohash to the Issuer here—instead, it should have a URI to look up on an issuer what key_id should be expected for each region, so it can verify the key is correct.

Reply:
Ah, that's much better!